### PR TITLE
About me

### PR DESCRIPTION
**Pull request by shegy28**
### Added About me section
**In this branch **

- Add about me section using Grid layout
- Used flexbox inside the grids to style the card
- Add skills and languages to the about me section using UL LI tags

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,30 +21,256 @@ <h3><a href="#">Asouzu Chimezie</a></h3>
...
-                                <li><img src="/images/Vector.svg" alt=""></li>
-                                <li><img src="/images/Vector (1).svg" alt=""></li>
-                                <li><img src="/images/Vector (2).svg" alt=""></li>
-                                <li><img src="/images/Vector (3).svg" alt=""></li>
-                                <li><img src="/images/Linkedin icon.svg" alt=""></li>
-                            </ul>
+                </div>
+                <div>
+                    <div class="card"> 
+                        <div class="card-img">
```
Comment (Kossi-stack) [inline index.html:91]: - [ ] Your images are not showing kindly add a dot before images "./images/.....

Comment (Kossi-stack) [reply index.html:91]: - [ ] Kindly add alt to all your images don't leave them empty.

File: index.html [hunk 2]
```diff
@@ -20,30 +21,256 @@ <h3><a href="#">Asouzu Chimezie</a></h3>
...
+                        <div class="card-info">
+                            <div class="info-header">
+                                <h4 class="tonic">Multi-Post
+                                    Stories
+                                </h4>
+                            </div>
+                            <div class="info-icons">
+                                <P class="info-icons-p1">CANOPY</P>
+                                <img src="/images/Counter.png" alt="">
+                                <p class="info-icons-p2">Back End Dev</p>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly use a header tag hx, for all project titles (e.g., "Tonic").

File: index.html [hunk 3]
```diff
@@ -20,30 +21,256 @@ <h3><a href="#">Asouzu Chimezie</a></h3>
...
+                            <img src="/images/Snapshoot Portfolio (2).png" alt="">
+                        </div>
+                        <div class="card-info">
+                            <div class="info-header">
+                                <h4 class="tonic">Tonic</h4>
+                            </div>
+                            <div class="info-icons">
+                                <P class="info-icons-p1">CANOPY</P>
+                                <img src="/images/Counter.png" alt="">
+                                <p class="info-icons-p2">Back End Dev</p>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly use a header tag hx, for all project titles (e.g., "Tonic").

File: index.html [hunk 4]
```diff
@@ -20,30 +21,256 @@ <h3><a href="#">Asouzu Chimezie</a></h3>
...
+                            <h1>About
+                                Myself
+                            </h1>
+                        </div>
+                        <div class="info">
+                            <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                        </div>
+                        <div>
+                            <div class="socials">
+                                <p>LET'S CONNECT</p>
```
Comment (Kossi-stack) [inline index.html]: -  [ ] Kindly use a header tag `hx`

File: index.html [hunk 5]
```diff
@@ -20,30 +21,256 @@ <h3><a href="#">Asouzu Chimezie</a></h3>
...
+                            </div>
+                            <div class="about-icons">
+                                <ul>
+                                    <li><img src="/images/Vector (1).svg" alt=""></li>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly let all the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @shegy28,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (shegy28) [reply index.html]: @Kossi-stack I used the h4 tag for the project titles already  (e.g., "Tonic"). what you highlighted  (e.g... "Canopy" in line 101) is not a project title  but text under the project title

Comment (shegy28) [reply index.html]: Oh i noticed what you talked about and  thanks for the helpful feedback i have updated them to the relevant hx tags,

**Review by anagudelogu — APPROVED** [review]
Hi @shegy28, 

You did a really nice job with your UI. Keep it up! 🍾 🎆 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



## Highlights
✅ Great and responsive UI design. 💯 
✅ Linters are passing. 👍🏼 
✅ Clean code. 🥇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**